code-of-conduct-languages/hi.md

```diff
@@ -0,0 +1,30 @@
+## CNCF सामुदायिक आचार संहिता v1.0
+
+### आचार संहिता का योगदानकर्ता
+
+इस परियोजना के योगदानकर्ताओं और अनुरक्षकों के रूप में, और एक खुले और स्वागत करने वाले समुदाय को बढ़ावा देने के हित में, हम उन सभी लोगों का सम्मान करने की प्रतिज्ञा करते हैं जो रिपोर्टिंग issues के माध्यम से योगदान करते हैं, फीचर अनुरोध पोस्ट करते हैं, प्रलेखन अपडेट करते हैं, pull requset या patches, और अन्य गतिविधियों में भाग लेते हैं।
+
+हम इस परियोजना में सभी के लिए उत्पीड़न-मुक्त अनुभव बनाने के लिए प्रतिबद्ध हैं, भले ही अनुभव के स्तर, लिंग, लिंग पहचान और अभिव्यक्ति, यौन अभिविन्यास, विकलांगता, व्यक्तिगत उपस्थिति, शरीर का आकार, नस्ल, जातीयता, आयु, धर्म, या राष्ट्रीयता।
+
+प्रतिभागियों द्वारा अस्वीकार्य व्यवहार के उदाहरणों में शामिल हैं:
+
+* लैंगिक भाषा या कल्पना का उपयोग
+* व्यक्तिगत हमले
+* ट्रोलिंग or अपमानजनक/derogatory टिप्पणी
+* सार्वजनिक या निजी उत्पीड़न
+* अन्य निजी जानकारी, जैसे कि भौतिक या electronic addresses, बिना स्पष्ट अनुमति के प्रकाशित करना
+* अन्य unethical या अव्यवसायिक आचरण।
+
+प्रोजेक्ट मेंटेनर के पास टिप्पणियों को हटाने, संपादित, या कोड, विकी संपादन, मुद्दे और अन्य योगदान अस्वीकार करने का अधिकार और जिम्मेदारी है, जो इस आचार संहिता में संरेखित नहीं हैं। इस आचार संहिता को अपनाने से, परियोजना के रखवाले इस परियोजना के प्रबंधन के हर पहलू के लिए इन सिद्धांतों को उचित और लगातार लागू करने के लिए खुद को प्रतिबद्ध करते हैं। परियोजना अनुचर जो आचार संहिता का पालन नहीं करते हैं या लागू नहीं करते हैं, उन्हें परियोजना टीम से स्थायी रूप से हटाया जा सकता है।
+
+यह आचार संहिता परियोजना के रिक्त स्थान और सार्वजनिक स्थानों पर लागू होती है जब कोई व्यक्ति परियोजना या उसके समुदाय का प्रतिनिधित्व करता है।
+
+कुबरनेट्स में अपमानजनक, उत्पीड़न या अन्यथा अस्वीकार्य व्यवहार के उदाहरणों को <आचरण@kubernetes.io> के माध्यम से [कुबरनेट्स आचार संहिता समिति](https://git.k8s.io/community/committee-code-of-conduct) से संपर्क करके सूचित किया जा सकता है। अन्य परियोजनाओं के लिए, कृपया किसी सीएनसीएफ प्रोजेक्ट मेंटेनर या हमारे मध्यस्थ, मिशी चौधरी <mishi@linux.com> से संपर्क करें।
```

Comment:
You can keep <आचरण@kubernetes.io> as <conduct@kubernetes.io>.

Reply:
Sure, will update that.